infra/Pos/DHT/Real/Param.hs

```diff
@@ -0,0 +1,17 @@
+module Pos.DHT.Real.Param
+       ( KademliaParams(..)
+       ) where
+
+import           Universum
+import           Pos.Util.TimeWarp   (NetworkAddress)
+import           Pos.DHT.Model.Types (DHTKey, DHTNode)
+
+-- | Parameters for the Kademlia DHT subsystem.
+data KademliaParams = KademliaParams {
+      kpIpPort          :: !NetworkAddress
+    , kpPeers           :: ![DHTNode]      -- ^ Peers passed from CLI
+    , kpKey             :: !(Maybe DHTKey)
+    , kpExplicitInitial :: !Bool
+    , kpDump            :: !FilePath       -- ^ Path to kademlia dump file
+    }
```

Comment:
I wonder why there's no `Show` instance -- pretty sure we'd like to print the config if encounter a fatal error (say, on initialization -- can't bind because wrong host is specified, etc).